docs/how-to/openstack-runner.md

```diff
@@ -44,16 +44,16 @@ clouds:
 
 The `clouds.yaml` documentation is [here](https://docs.openstack.org/python-openstackclient/pike/configuration/index.html#clouds-yaml).
 
-### OpenStack Flavor
+### OpenStack Flavour
```

Comment:
The style guide recommends using British English, so that's why I made the change. I appreciate that some external docs and some programming functions use American English, and thus our docs will differ in spelling. In any event, I think it's best to adhere to the style guide.